content/authors/revanmuhammad/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+# Display name
+name: Revan Muhammad Dafa
+
+# To be displayed on the document's title that it shown in browser's title bar
+title: Revan Muhammad Dafa
+
+# Username (this should match the folder name)
+authors:
+- revanmuhammad
+
+# Role/position
+role: Student
+
+# Organizations/Affiliations
+organizations:
+- name: Sriwijaya University
+  url: "https://unsri.ac.id/"
+
+# Short bio (displayed in user profile at end of posts)
+bio: Student in Sriwijaya University
+
+interests:
+- Machine Learning
+- Software Engineering
+- Web Developer
```

Comment:
It's better to write it as Web Development.

Reply:
Oke kak thanks for the correction

Reply:
Okeeeee kak, thanks for the correction